comps/comps-katello-pulp-server-rhel6.xml

```diff
@@ -23,10 +23,6 @@
        <packagereq type="default">pulp-consumer-client</packagereq>
        <packagereq type="default">pulp-docker-plugins</packagereq>
        <packagereq type="default">pulp-docker-admin-extensions</packagereq>
-       <packagereq type="default">pulp-nodes-admin-extensions</packagereq>
-       <packagereq type="default">pulp-nodes-child</packagereq>
```

Comment:
If we install a full pulp vs pulp::child what gets configured wrong? As in, can we identify that easily to correct or is this 3.1 technical debt?